.github/workflows/chromium-validation.yaml

```diff
@@ -0,0 +1,33 @@
+name: Validate Latest Chromium Binary PDF generation
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-unit-tests:
+    name: Run Unit Tests with Latest Chrome
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+
+      - name: Download and install latest Chrome
+        run: |
+          wget -q -O chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+          sudo apt install -y ./chrome.deb
```

Comment:
For uniformity with how we're installing Chromium in the [Dockerfile](https://github.com/CCI-MOC/invoicing/blob/7d78602b7e7a99adee4f848211ad64537b111e35/Dockerfile#L5), please use the same method.